netdata-installer.sh

```diff
@@ -288,7 +293,7 @@ while [ -n "${1}" ]; do
     "--use-system-lws") USE_SYSTEM_LWS=1 ;;
```

Comment:
This line can be dropped, it’s left-over from the legacy ACLK code and got missed during that cleanup.